tests/managed_services/mas_debezium/test_mas_debezium.py

```diff
@@ -0,0 +1,30 @@
+import shlex
+
+import pytest
+from constants import TEST_RECORD
+
+
+pytestmark = pytest.mark.mas_debezium
+
+
+class TestDebezium:
+    def test_kafka_topics(self, kafka_record, kafka_sa_acl, consumer_pod):
+        """
+        Test for managed kafka resources setup,
+        usage and teardown via rhoas sdk
+        """
+        # Get consumed record from test topic via kcat pod
```

Comment:
this comment should be removed; the code should be self-explanatory

Reply:
done